src/ci/docker/dist-various-2/Dockerfile

```diff
@@ -65,6 +71,8 @@ ENV TARGETS=$TARGETS,wasm32-unknown-unknown
 ENV TARGETS=$TARGETS,x86_64-sun-solaris
 ENV TARGETS=$TARGETS,x86_64-unknown-linux-gnux32
 ENV TARGETS=$TARGETS,x86_64-unknown-cloudabi
+ENV TARGETS=$TARGETS,x86_64-fortanix-unknown-sgx
 
 ENV RUST_CONFIGURE_ARGS --enable-extended --enable-lld --disable-docs
-ENV SCRIPT python2.7 ../x.py dist --target $TARGETS
+ENV SCRIPT python2.7 ../x.py dist --target $TARGETS && \
+         cp /x86_64_fortanix_unknown_sgx/lib/* /checkout/obj/build/x86_64-unknown-linux-gnu/stage2/lib/rustlib/x86_64-fortanix-unknown-sgx/lib/
```

Comment:
Can this be done by rustbuild instead of having to manually add it here?

Reply:
Could you provide any pointers on how to do this?

Reply:
You can look around rustbuild for things like rsend.o and mingw, that platform and musl already copy files around like this.

Reply:
In my current push, I have followed the `copy_thrid_party_objects` logic for `musl`.
For this target, the Docker build process sets and ENV variable indicating source path of the libunwind.a. The boot strap code reads the same and copies the file to correct destination.